src/Symfony/Bundle/FrameworkBundle/Resources/config/message.xml

```diff
@@ -0,0 +1,59 @@
+<?xml version="1.0" ?>
+
+<container xmlns="http://symfony.com/schema/dic/services"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
+
+    <services>
+        <defaults public="false" />
+
+        <!-- Bus -->
+        <service id="message_bus" class="Symfony\Component\Message\MessageBus" public="true">
+            <argument type="collection" /> <!-- Middlewares -->
+        </service>
+
+        <service id="Symfony\Component\Message\MessageBusInterface" alias="message_bus" />
+
+        <!-- Handlers -->
+        <service id="message.handler_resolver" class="Symfony\Component\Message\HandlerLocator">
+            <argument type="collection" /> <!-- Message to handler mapping -->
+        </service>
+
+        <service id="message.middleware.call_message_handler" class="Symfony\Component\Message\Middleware\HandleMessageMiddleware">
+            <argument type="service" id="message.handler_resolver" />
+
+            <tag name="message_middleware" priority="-10" />
```

Comment:
I wonder if `message_middleware` is the best choice possible for this tag name. Unlike other frameworks, in Symfony we don't use the "middleware" word anywhere (not even in docs).

Reply:
Correct, we don't use middlewares yet. But I believe the term starts to be widely known and the good is: they mean the same everywhere :)